include/dxc/dxcapi.internal.h

```diff
@@ -196,6 +196,11 @@ struct IDxcLangExtensions2 : public IDxcLangExtensions {
   virtual HRESULT STDMETHODCALLTYPE SetTargetTriple(LPCSTR name) = 0;
 };
 
+CROSS_PLATFORM_UUIDOF(IDxcPdbUtils2, "7D678438-A9F8-493F-861C-4DA9366898FE")
+struct IDxcPdbUtils2 : public IUnknown {
```

Comment:
I don't think we have to rev the interface.  Let's add to the existing one, since it was not yet considered final.